src/expint.jl

```diff
@@ -297,29 +297,60 @@ end
 # series about origin, general ν
 # https://functions.wolfram.com/GammaBetaErf/ExpIntegralE/06/01/04/01/01/0003/
 function En_expand_origin_general(ν::Number, z::Number)
-    gammaterm = En_safe_gamma_term(ν, z)
+    # gammaterm = En_safe_gamma_term(ν, z)
+    gammaterm = gamma(1-ν)*z^(ν-1)
     frac = one(real(z))
-    sumterm = frac / (1 - ν)
+    blowup  = abs(1 - ν) < 0.5 ? frac / (1 - ν) : zero(real(z))
+    sumterm = abs(1 - ν) < 0.5 ? zero(real(z)) : frac / (1 - ν)
     k, maxiter = 1, 100
-    ϵ = 10*eps(real(sumterm))
+    ϵ = 10*eps(typeof(real(sumterm)))
```

Comment:
?

Reply:
This should be an absolute precision, not a relative precision, so I think the old code was correct.